BeagleBone_black/motor_control/motor_controller.hpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Author: Sean Mullan and Jack Horsburgh
+ * Organisation: HYPED
+ * Date: 17/02/18
+ * Description:
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef BEAGLEBONE_BLACK_MOTOR_CONTROL_MOTOR_CONTROLLER_HPP_
+#define BEAGLEBONE_BLACK_MOTOR_CONTROL_MOTOR_CONTROLLER_HPP_
+
+#include "motor.hpp"
+
+class MotorController {
+ public:
+   MotorController();
+   void setupMotors();
+   void accelerateMotors();
+   void decelerateMotors();
+   void stopMotors();
+   int calculateAccelerationRPM(double translational_velocity);
+   int calculateDecelerationRPM(double translational_velocity);
+ private:
```

Comment:
See [class format](https://hyp-ed.github.io/styleguide/#Class_Format) section in the style guide. There should be an empty line before this `private:`. Also correct the indentation, declarations should be indented 2 spaces relative to `class` (you have 3).